INSTALL.md

```diff
@@ -72,17 +72,12 @@ bundle exec rake ts:configure ts:start
 
 ## Asset compilation
 
-On development, assets are compiled on automatically everytime a page is requested to rails. To avoid the uptime we recommend running webpack-dev-server in a second terminal:
+In development, assets are compiled automatically when a page is requested from rails. To allow online updates when an asset file changes, start:
```

Comment:
I don't think I get the "online" updates part. According to rails documentation the right term is _live code reloading_. So maybe:

```suggestion
In development, assets are compiled automatically on demand when a Rails page loads. If you want to use [live code reloading](https://guides.rubyonrails.org/webpacker.html#running-webpacker-in-development) you'll need to run:
```